### DISCUSSION TITLE
How do I access the API?

### DISCUSSION THREAD

**Discussion by ThendilMC** [SchnTgaiSpock/Gastronomicon#16] [Q&A]
Some information on how to access and use the API in layman terms would be really helpful.

**Comment by SchnTgaiSpock** [comment]
there are api setup instructions here https://schn.pages.dev/gastronomicon/api-setup

some examples:
- https://github.com/SchnTgaiSpock/Wildernether/blob/master/pom.xml#L113
- https://github.com/ProfElements/DynaTech/blob/main/pom.xml#L123
